tests/Mapper/Builder/OptionsBuilderTest.php

```diff
@@ -48,6 +57,9 @@ public function testCreateOneToMany(): void
         ], $builder->getOptions());
     }
 
+    /**
+     * @group legacy
```

Comment:
Please, add a `NEXT_MAJOR` comment explaining what should we do with these legacy tests in the next major release.